packages/circuit-ui/components/TopNavigation/components/ProfileMenu/ProfileMenu.tsx

```diff
@@ -126,7 +136,11 @@ function Profile({
       title={profileLabel}
       isActive={isOpen || profileIsActive}
     >
-      <UserAvatar {...userAvatar} variant="identity" />
+      {userAvatar ? (
+        <UserAvatar {...userAvatar} variant="identity" />
+      ) : (
+        <AvatarPlaceholder />
+      )}
```

Comment:
You could also pass the custom placeholder to the Avatar component by passing the svg placeholder as a fallback for `userAvatar`, just like in the Avatar's implementation ([source](https://github.com/sumup-oss/circuit-ui/blob/6a30fd27dbfb63b73af42fefe336d90ae037e4e0/packages/circuit-ui/components/Avatar/Avatar.tsx#L87))

```tsx
// note the data:image/svg 
const placeholder = 'data:image/svg+xml;utf8,<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M22.981 21.22A11.074 11.074 0 0012.001 12a11.074 11.074 0 00-10.982 9.22A1.576 1.576 0 002.6 23h18.8a1.577 1.577 0 001.581-1.78zm-10.98-1.72a1.5 1.5 0 110-3 1.5 1.5 0 010 3zm4.5-14a4.5 4.5 0 11-9 0 4.5 4.5 0 019 0z" fill="currentColor"/></svg>';

const Example = () => (
  <UserAvatar src={userAvatar.src || placeholder} {...userAvatar} variant="identity" />
);
```

Reply:
Unfortunately not. Here's the approach you suggested:

<img width="210" alt="Screenshot 2021-08-12 at 09 35 33" src="https://user-images.githubusercontent.com/11017722/129156800-025e41c5-8f15-4c3b-9786-d5f0e2b92a93.png">

Here's the design:

<img width="284" alt="Screenshot 2021-08-12 at 09 36 22" src="https://user-images.githubusercontent.com/11017722/129156868-9a8e3a2c-357f-4c0c-ac2b-e728aad0041a.png">

It also wouldn't be possible to color the SVG blue when active.

Reply:
Aaah the background is handled by the component, true. Then all good 🙂